Example/Podfile.lock

```diff
@@ -85,9 +84,9 @@ EXTERNAL SOURCES:
   Artsy+UIFonts:
     :git: https://github.com/artsy/Artsy-UIFonts.git
   Emission:
-    :path: ../
+    :path: "../"
   React:
-    :path: ../node_modules/react-native
+    :path: "../node_modules/react-native"
```

Comment:
Right, I believe Eigen should be possible with system Ruby, but you have to do `bundle install --without development` so that the push notification gem doesn’t get installed (which requires newer Ruby).